data_rentgen/server/settings/auth/__init__.py

```diff
@@ -0,0 +1,29 @@
+# SPDX-FileCopyrightText: 2024 MTS PJSC
+# SPDX-License-Identifier: Apache-2.0
+from pydantic import BaseModel, Field, ImportString
+
+
+class AuthSettings(BaseModel):
```

Comment:
New settings classes should be documented, probably in a separated PR

Reply:
Yes. In text PR with Keycloak provider I am add all docs at once.